src/main/java/io/stargate/sgv2/jsonapi/config/constants/LoggingConstants.java

```diff
@@ -0,0 +1,11 @@
+package io.stargate.sgv2.jsonapi.config.constants;
```

Comment:
@Yuqi-Du Why we need this?

Reply:
Yes, there is one constant used somewhere.
Others are redundant, these are just constant, can delete later